contracts/TranscodeJobs.sol

```diff
@@ -63,6 +66,8 @@ library TranscodeJobs {
         self.jobs[self.numJobs].transcoderAddress = _electedTranscoder;
         self.numJobs++;
 
+        NewJob(_electedTranscoder, msg.sender);
```

Comment:
Interesting, so if the event is defined in both the main contract and the imported library, then are they considered to be the same if they have the same signature? Or does subscribing to one not necessarily subscribe you to the other?